session/session.go

```diff
@@ -1226,6 +1226,10 @@ func createSessionFunc(store kv.Storage) pools.Factory {
 		if err != nil {
 			return nil, errors.Trace(err)
 		}
+		err = se.sessionVars.SetSystemVar(variable.TiDBEnableWindowFunction, "1")
```

Comment:
There is `DefEnableWindowFunction`, probably use that.

Reply:
Yes, I updated.